doc/classes/DisplayServer.xml

```diff
@@ -1120,6 +1127,14 @@
 				Sets the window icon (usually displayed in the top-left corner) in the operating system's [i]native[/i] format. The file at [param filename] must be in [code].ico[/code] format on Windows or [code].icns[/code] on macOS. By using specially crafted [code].ico[/code] or [code].icns[/code] icons, [method set_native_icon] allows specifying different icons depending on the size the icon is displayed at. This size is determined by the operating system and user preferences (including the display scale factor). To use icons in other formats, use [method set_icon] instead.
 			</description>
 		</method>
+		<method name="set_system_theme_change_callback">
+			<return type="void" />
+			<param index="0" name="callable" type="Callable" />
+			<description>
+				Sets the [param callable] that should be called when system theme settings are changed. Callback method should have zero arguments.
+				[b]Note:[/b] This method is implemented on Android, iOS, macOS, Windows, and Linux (X11).
```

Comment:
```suggestion
				[b]Note:[/b] This method is implemented on Android, iOS, macOS, Windows, and Linux (X11 and Wayland).
```

Reply:
Actually should be `(X11/Wayland)`, there's only 2 occurrences of `(X11 and Wayland)` which would also be worth changing to keep it consistent.
```
$ rg "Linux \(X11/Wayland\)" | wc -l
35
$ rg "Linux \(X11 and Wayland\)" | wc -l
2
```